pkg/skaffold/schema/v2beta9/upgrade_test.go

```diff
@@ -0,0 +1,186 @@
+/*
+Copyright 2020 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v2beta9
+
+import (
+	"testing"
+
+	yaml "gopkg.in/yaml.v2"
+
+	next "github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/latest"
+	"github.com/GoogleContainerTools/skaffold/testutil"
+)
+
+func TestUpgrade(t *testing.T) {
+	yaml := `apiVersion: skaffold/v2beta9
```

Comment:
Please add a `port-forward` config with int port so in next PR, you can verify it gets updated correctly.

Reply:
I will make this change in #5009